themes/mlibrary_new/exhibit-builder/exhibits/tags.php

```diff
@@ -1,5 +1,6 @@
   <div class="col-sm-12 tag-cloud">
      <li>Browse by Popular Tags</li>
-  </div>
```

Comment:
I encourage avoiding whitespace-only changes in the same commits as functional changes.

Reply:
I am sorry I did not get this part?